docs/units/standard_units.rst

```diff
@@ -1,3 +1,5 @@
+.. _doc_standard_units:
```

Comment:
Why the `doc_` preceding the actual name? Wouldn't it be handier to just call it `_standard_units? That seemed to be the standard in time at least... (I do think it is good to add references!).